docs/src/user-guide/core-unstructured/clp.md

```diff
@@ -105,25 +105,36 @@ Currently, timestamps must be specified as milliseconds since the UNIX epoch.
 
 # Parallel Compression
 
-By default, `clp` uses an embedded SQLite database, so each directory containing archives can only
-be accessed by a single `clp` instance.
+To enable parallel compression to the same archives directory, `clp` (and by extension, `clg`) needs
+to be configured to use a MySQL-type database (e.g., MariaDB) rather than the default---an embedded
+SQLite database (which doesn't support concurrent writes).
 
-To enable parallel compression to the same archives directory, `clp`/`clg` can be configured to use
-a MySQL-type database (e.g., MariaDB) as follows:
+:::{warning}
+Running multiple `clp` instances with SQLite can fail due to the error "database is locked".
+:::
+
+You can configure `clp` and `clg` to use a MySQL-type database as follows:
```

Comment:
ditto - if you think "compatible" is a better word for those who don't know MySQL & MariaDB
```suggestion
You can configure `clp` and `clg` to use a MySQL-compatible database as follows:
```